__tests__/app.test.js

```diff
@@ -438,3 +455,29 @@ test("Status 404 path not found, if review_id entered is valid but does not exis
       expect(msg).toBe("Path not found");
     });
 });
+
+test("Delete /api/comments/:comment_id Status 204 by comment_id", () => {
+  return request(app)
+    .delete("/api/comments/2")
+    .expect(204)
+    .then(({ body }) => {
+      expect(body).toEqual({});
+    });
+});
+
+test("status 400 Bad request when comment is not valid data type", () => {
+  return request(app)
+    .delete("/api/comments/notacomment")
+    .expect(400)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe("Bad request");
+    });
+});
+test("status 400 Bad request when comment is not valid data type", () => {
+  return request(app)
```

Comment:
It is a valid data type at this point - it just does not exist so make sure your description reflects -> please do not copy and paste, this is how small but effective errors are created.

Reply:
Changed test to make more clear